airbyte-integrations/connectors/source-hellobaton/source_hellobaton/manifest.yaml

```diff
@@ -0,0 +1,171 @@
+    type: HttpRequester
+    url_base: https://{{ config['company'] }}.hellobaton.com/api/
+    path: '{{ parameters.path }}'
+    http_method: GET
+    request_parameters: {}
+    request_headers: {}
+    authenticator:
+      type: ApiKeyAuthenticator
+      api_token: "{{ config['api_key'] }}"
+      inject_into:
+        type: RequestOption
+        inject_into: request_parameter
+        field_name: api_key
+    request_body_json: {}
+
+  retriever:
+    type: SimpleRetriever
+    requester:
+      $ref: "#/definitions/requester"
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      $ref: "#/definitions/paginator"
+
+  base_stream:
+    type: DeclarativeStream
+    # name: '{{ parameters.name }}'
+    primary_key:
+      - id
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/retriever"
+
+  activity_stream:
+    $ref: "#/definitions/base_stream"
+    name: "activity"
+    $parameters:
+      path: "activity"
+
+  companies_stream:
+    $ref: "#/definitions/base_stream"
+    name: "companies"
+    $parameters:
+      path: "companies"
+
+  milestones_stream:
+    $ref: "#/definitions/base_stream"
+    name: "milestones"
+    $parameters:
+      path: "milestones"
+
+  phases_stream:
+    $ref: "#/definitions/base_stream"
+    name: "phases"
+    $parameters:
+      path: "phases"
+
+  project_attachments_stream:
+    $ref: "#/definitions/base_stream"
+    name: "project_attachments"
+    $parameters:
+      path: "project_attachments"
+
+  projects_stream:
+    $ref: "#/definitions/base_stream"
+    name: "projects"
+    $parameters:
+      path: "projects"
+
+  tasks_stream:
+    $ref: "#/definitions/base_stream"
+    name: "tasks"
+    $parameters:
+      path: "tasks"
+
+  task_attachments_stream:
+    $ref: "#/definitions/base_stream"
+    name: "task_attachments"
+    $parameters:
+      path: "task_attachments"
+
+  templates_stream:
+    $ref: "#/definitions/base_stream"
+    name: "templates"
+    $parameters:
+      path: "templates"
+
+  time_entries_stream:
+    $ref: "#/definitions/base_stream"
+    name: "time_entries"
+    $parameters:
+      path: "time_entries"
+
+  users_stream:
+    $ref: "#/definitions/base_stream"
+    name: "users"
+    $parameters:
+      path: "users"
+
+streams:
+  - "#/definitions/activity_stream"
+  - "#/definitions/companies_stream"
+  - "#/definitions/milestones_stream"
+  - "#/definitions/phases_stream"
+  - "#/definitions/project_attachments_stream"
+  - "#/definitions/projects_stream"
+  - "#/definitions/tasks_stream"
+  - "#/definitions/task_attachments_stream"
+  - "#/definitions/templates_stream"
+  - "#/definitions/time_entries_stream"
+  - "#/definitions/users_stream"
+
+spec:
+  documentation_url: https://docs.airbyte.com/integrations/sources/baton
+  type: Spec
+  connection_specification:
+    $schema: http://json-schema.org/draft-07/schema#
+    type: object
+    additionalProperties: false
```

Comment:
can you set this to true and skip backward incompatibility check in the acceptance-test-config.yml for the spec task?